recipes/botan/all/conanfile.py

```diff
@@ -0,0 +1,320 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanException, ConanInvalidConfiguration
+from conans.model.version import Version
+
+
+class BotanConan(ConanFile):
+    name = 'botan'
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/randombit/botan"
+    license = "BSD 2-clause"
```

Comment:
```suggestion
    license = "BSD-2-Clause"
```

follow the official short description https://spdx.org/licenses/BSD-2-Clause.html